src/metatensor/models/cli/eval_model.py

```diff
@@ -35,7 +37,7 @@ def _add_eval_model_parser(subparser: argparse._SubParsersAction) -> None:
     parser.set_defaults(callable="eval_model")
     parser.add_argument(
         "model",
-        type=load_model,
+        type=torch.jit.load,
```

Comment:
Maybe this is a rookie question, but the exported model here has a `torch.jit.load` type? Why it's not `str` (for a filename type), or a `RecursiveScriptModel`?

Reply:
Ok, I see, thank you!